src/content/docs/errors-inbox/browser-tab.mdx

```diff
@@ -0,0 +1,147 @@
+
+import errorsinboxErrorsInboxTriageSection from 'images/errors-inbox_screenshot-crop_errors-inbox-triage-section.webp'
+
+import errorsinboxEventTrailforErrorsInbox from 'images/errors-inbox_screenshot-crop_event-trail-for-errors-inbox.webp'
+
+import errorsinboxErrorsInboxStackTrace from 'images/errors-inbox_screenshot-crop_errors-inbox-stack-trace.webp'
+
+With **Group errors** tab you can dynamically filter and group errors for deeper analysis.
+
+<img
+    title="browser group errors tab "
+    alt="a screenshot depiciting the browser group errors tab"
+    src={errorsinboxBrowserGroupErrorsTab}
+/>
+
+Group errors shows you where your JavaScript errors are happening, and provides tools to help you figure out the root cause. You can also query and create dashboards of JS error data in the [query builder](/docs/query-your-data/explore-query-data/query-builder/introduction-query-builder), or use the browser API to monitor handled errors.
+To examine JavaScript errors:
+1. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Browser > (select an app) > Errors**.
+2. Use the [available options in the UI](#ui-features) to examine what may be causing the errors.
+3. Optional: [Review event trail ](#event-trail) or [stack traces](#stack-trace) in the UI.
+4. Optional: [Query and create dashboards for JS error data](#insights) with the query builder, or [use the API](#api-logs) to monitor or log errors.
+
+## Errors list view  [#list-view]
+
+Start with the page loads errors chart to see at a glance whether there are any unexpected spikes, dips, or patterns with errors in general.
+Correlate any general patterns on the top 5 errors chart to alerts occurring during the same time period.
+* **Dynamic grouping:** The default grouping for error occurrences is based on the error message. You have the option to change the grouping options by any attribute, up to five attributes at a time.
+* **Filtering:** Many New Relic customers instrument [custom attributes](/docs/data-apis/custom-data/custom-events/collect-custom-attributes/#enabling-custom). Filtering on a specific custom attribute can be a quick way to cut through the noise of all error occurrences. 
+
+<img
+    title="browser group errors list view"
+    alt="A screenshot depicting the browser group errors list view. "
+    src={errorsinboxBrowserErrorsListView}
+/>
+
+## Error profiles [#JS-profiles] 
+
+Error profiles can help you answer questions such as: Which users are most common across all the error events in the selected time window? or which session is most common across all the error events in the selected time window?
+You can download the attributes as CSV. You can also view the attributes breakdown when you select a row.
+
+<img
+    title="JS error profiles"
+    alt="A screenshot depicting error profiles for JS."
+    src={errorsinboxBrowserErrorProfiles}
+/>
```

Comment:
I'd add here the `figcaption` to indicate how you get to this screenshot, for example: Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Browser**. Then, click your app and under the **Errors** section, click **Errors (errors inbox)**. Click the **View profiles** button to open the attribute profiles table.